gateway/middleware.go

```diff
@@ -29,7 +29,10 @@ import (
 	"github.com/TykTechnologies/tyk/user"
 )
 
-const mwStatusRespond = 666
+const (
+	mwStatusRespond         = 666
+	defaultOrgSessionExpiry = int64(604800)
```

Comment:
am wonderin if we should write constants in uppercase

Reply:
converted to uppercase!